samples/deployment/test-network/setup.sh

```diff
@@ -23,70 +23,66 @@ backup() {
   fi
 }
 
+##############################################################################################
+# Get the fabric samples repo and fetch binaries and container images
+##############################################################################################
+
 echo "Prepare fabric samples test-network for FPC"
 
 FABRIC_SAMPLES=${FPC_PATH}/samples/deployment/test-network/fabric-samples
-CORE_PATH=${FABRIC_SAMPLES}/config/core.yaml
-DOCKER_PATH=${FABRIC_SAMPLES}/test-network/docker
-DOCKER_COMPOSE_TEST_NET=${DOCKER_PATH}/docker-compose-test-net.yaml
-DOCKER_COMPOSE_CA=${DOCKER_PATH}/docker-compose-ca.yaml
 
-if [ ! -d "${FABRIC_SAMPLES}/bin" ]; then
-  echo "Error: no fabric binaries found, see README.md"
-  exit 1
+if [ ! -d "${FABRIC_SAMPLES}" ]; then
+  echo "Fabric samples not found! cloning now!"
+  git clone https://github.com/hyperledger/fabric-samples "${FABRIC_SAMPLES}"
+  pushd "${FABRIC_SAMPLES}"
+  git checkout -b "works" 50b69f6
+  popd
 fi
 
-
-###############################################
-# Adding FPC support to core.yaml
-###############################################
-
-echo "Adding FPC external builder to core.yaml"
-# Create a backup copy of `core.yaml` first and always work from there.
-backup ${CORE_PATH}
-yq eval-all 'select(fileIndex == 0) * select(fileIndex == 1)' -i ${CORE_PATH} core_ext.yaml
+if [ ! -d "${FABRIC_SAMPLES}/bin" ]; then
+  echo "Error: no fabric binaries found. Let's run 'network.sh prereq'"
+  pushd "${FABRIC_SAMPLES}/test-network"
+  ./network.sh prereq
```

Comment:
Before deleting my already present `fabric_samples` folder, I run `./setup.sh` and the `bin folder` wasn't there but `./network.sh prereq` doesn't work if you run it

Reply:
@osamamagdy be careful because setup is now using a newer commit of fabric samples ....